orc8r/gateway/python/magma/common/BUILD.bazel

```diff
@@ -6,7 +6,7 @@
 # Unless required by applicable law or agreed to in writing, software
 # distributed under the License is distributed on an "AS IS" BASIS,
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
+# See the License for the specifi   c language governing permissions and
```

Comment:
I'm guessing this is not meant to be here :p

Reply:
no :/ - thank you